File: front/customer_app/lib/models/eval.dart
class Eval {
  final String customerId;
  final String assistantId;
  final String desc;
  final int points;
  Eval(
      {required this.customerId,
      required this.assistantId,
      required this.desc,
      required this.points});

  Eval.fromJson(Map<String, dynamic> json)
      : customerId = json['CustomerId'],
        assistantId = json['AssistantId'],
        desc = json['Description'],
        points = json['Points'];

  Map<String, dynamic> toJson() => {
        'CustomerId': customerId,
        'AssistantId': assistantId,
        'Description': desc,
        'Points': points,
      };
}

File: back/MobileApi/MobileApi/Dtos/Eval.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MobileApi.Dtos
{
    public class Eval
    {
        public Guid CustomerId { get; set; }
        public Guid AssistantId { get; set; }
        public string Description { get; set; }
        public int Points { get; set; }
    }
}
